src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -377,6 +377,10 @@ static bool ReferencesNServiceBus(Assembly assembly, List<AssemblyName> processe
                 {
                     continue;
                 }
+                catch (FileLoadException)
```

Comment:
I had to do the same in the host since missing asm redirects causes this. Was this the reason?